src/optimizer_interface.jl

```diff
@@ -921,21 +921,17 @@ julia> is_solved_and_feasible(model)
 false
 
 julia> assert_is_solved_and_feasible(model)
-ERROR: The model was not solved correctly. Here is a summary of the solution to help debug why this happened:
-
-* Solver : Ipopt
-
-* Status
-  Result count       : 0
-  Termination status : OPTIMIZE_NOT_CALLED
-  Message from the solver:
-  "optimize not called"
-
-* Candidate solution (result #1)
-  Primal status      : NO_SOLUTION
-  Dual status        : NO_SOLUTION
-
-* Work counters
+ERROR: The model was not solved correctly. Here is the output of `solution_summary` to help debug why this happened:
+
+solution_summary(; result = 1, verbose = false)
+├ solver_name          : Ipopt
+├ Solution quality
+│ ├ termination_status : OPTIMIZE_NOT_CALLED
+│ ├ result_count       : 0
+│ └ raw_status         : optimize not called
+└ Solution (; result = 1)
```

Comment:
The ";" seems a little misterious.
Maybe just (result = 1) or do it in a extra line result_index : 1